cares-center-inventory-Song/schemas.py

```diff
@@ -0,0 +1,141 @@
+from pydantic import BaseModel, validator
+from typing import Optional
+
+# Manager
+class ManagerBase(BaseModel):
+    manager_name: str
+    passcode: int
```

Comment:
Put passcode in ManagerCreate and Manager and remove it from ManagerBase. Currently you would have to supply the manager's passcode if you wanted to delete them, but we just want the name.